### PR TITLE
Droth 2511 and Droth 2512

### PR DESCRIPTION
**Pull request by RobertVisan**

### CODE REVIEW

File: digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+DELETE FROM SINGLE_CHOICE_VALUE WHERE ENUMERATED_VALUE_ID IN
+    (SELECT ID FROM ENUMERATED_VALUE WHERE NAME_FI IN
+        ('I1 Sulkupuomi', 'I2.1 Sulkuaita', 'I2.2 Sulkuaita nuolilla', 'I3.1 Sulkupylväs vasemmalla', 'I3.2 Sulkupylväs oikealla',
+         'I3.3 Sulkupylväs', 'I4 Sulkukartio', 'I12.1 Reunapaalu vasemmalla', 'I12.2 Reunapaalu oikealla'));
+
+DELETE FROM ENUMERATED_VALUE WHERE NAME_FI IN ('I1 Sulkupuomi', 'I2.1 Sulkuaita', 'I2.2 Sulkuaita nuolilla', 'I3.1 Sulkupylväs vasemmalla', 'I3.2 Sulkupylväs oikealla',
+'I3.3 Sulkupylväs', 'I4 Sulkukartio', 'I12.1 Reunapaalu vasemmalla', 'I12.2 Reunapaalu oikealla');
+
+UPDATE ENUMERATED_VALUE SET NAME_FI = 'C42 Taksin pysähtymispaikka' WHERE NAME_FI = 'C42 Taksin pysäyttämispaikka';
```
Comment (marquesfm) [inline digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql:15]: Can we put some comments were to understand what is done in each line and when starts the fixes for specifics Bugs/Tasks?

**Review by marquesfm — COMMENTED** [review]
OK with a simple suggestion.

File: digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql (hunk 1 above)
Comment (RobertVisan) [reply digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql:20]: sql code described for each task

Comment (marquesfm) [reply digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql:20]: OK

**Review by FilPinto — APPROVED** [review]

File: digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+-- For DROTH-2512
```
Comment (FilPinto) [inline digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql:1]: Change file name from V233 to V234

Comment (RobertVisan) [reply digiroad2-oracle/src/main/resources/db/migration/V233__remove_unnecessary_traffic_signs_and_name_updates.sql:1]: Yes I will do that

**Pull request merged**